diagnostic.md

```diff
@@ -57,13 +95,19 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
+//Running out of time
 
+$(document).ready(function(){
+    $("li").parentsUntil("body").hide();
```

Comment:
This will only select ancestor elements until it reaches `<body>` but not actually select it.  The only elements that will be hidden will be `<ol>` and its descendants.